cars/carsRouter.js

```diff
@@ -0,0 +1,25 @@
+const express = require("express");
+const router = express.Router();
+const db = require("../data/db-config");
+
+router.get("/", async (req, res) => {
+  const cars = await db("cars");
```

Comment:
I believe these should live in the try statement. If this fails to fetch cars does it default to the catch block? That's something I'd have to test out.